### ISSUE TITLE
Cue points for individual words

### ISSUE THREAD

**Issue by Rillke** [abhirooptalasila/AutoSub#14]
YouTube's latest speech recognition creates cue points for individual words which are made visible in the moment, they are spoken. 

YT is transmitting their subtitles in a format that looks like this (which I do not recognize):

```json
{
  "wireMagic": "pb3",
  "pens": [ {
  
  } ],
  "wsWinStyles": [ {
  
  }, {
    "mhModeHint": 2,
    "juJustifCode": 0,
    "sdScrollDir": 3
  } ],
  "wpWinPositions": [ {
  
  }, {
    "apPoint": 6,
    "ahHorPos": 20,
    "avVerPos": 100,
    "rcRows": 2,
    "ccCols": 40
  } ],
  "events": [ {
    "tStartMs": 0,
    "dDurationMs": 2795440,
    "id": 1,
    "wpWinPosId": 1,
    "wsWinStyleId": 1
  }, {
    "tStartMs": 80,
    "dDurationMs": 3119,
    "wWinId": 1,
    "segs": [ {
      "utf8": "hey",
      "acAsrConf": 255
    }, {
      "utf8": " everybody",
      "tOffsetMs": 160,
      "acAsrConf": 255
    }, {
      "utf8": " how's",
      "tOffsetMs": 480,
      "acAsrConf": 255
```

However, [VTT subtitle format](https://developer.mozilla.org/en-US/docs/Web/API/WebVTT_API#cue_payload_text_tags) also supports cues for individual words (c.f. Karaoke style text), although not yet supported natively by the Firefox I tested in.

Timing information is available through DeepSpeeche's [`sttWithMetadata()`](https://deepspeech.readthedocs.io/en/latest/Python-API.html#native_client.python.Model.sttWithMetadata) and is easily transformed from character timings to word timings using their [client.py](https://deepspeech.readthedocs.io/en/latest/Python-Examples.html).

The request is to have an output file that allows us to do it like YT: Display individual words as they are spoken.

**Comment by abhirooptalasila** [comment]
Hi
Yes, I've come across this. But I didn't think anyone would be needing a JSON output of the individual candidate transcripts. You can have a look at the source code [here](https://deepspeech.readthedocs.io/en/v0.9.3/_modules/native_client/python.html). I can't work on this right now, but please do raise a PR incase you work on this.

**Comment by DerrickGibbs1** [comment]
Implemented in #24
